src/components/Sum/Sum.jsx

```diff
@@ -1 +1,3 @@
-// export const Sum = () => ();
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
```

Comment:
Please, edit this section according to the checklist)
2. [CODE STYLE] - Use string interpolation inside tag content.